src/gc/config.d

```diff
@@ -16,7 +16,7 @@ struct Config
 {
     bool disable;            // start disabled
     ubyte profile;           // enable profiling with summary when terminating program
-    string gc = "conservative"; // select gc implementation conservative|manual
+    string gc = "precise"; // select gc implementation conservative|precise|manual
```

Comment:
Do we really want to change that default?

Reply:
No, but I want the tests to run with the precise version. It should be reverted to "conservative" before merging.